lib/document/templates.js

```diff
@@ -124,17 +124,33 @@ ${faDom.css()}
 }
 
 const syntaxHighlighterHead = (node, syntaxHighlighter, attrs) => {
+  let header = ''
   if (syntaxHighlighter !== Opal.nil && syntaxHighlighter['$docinfo?']('head')) {
-    return syntaxHighlighter['$docinfo']('head', node, Opal.hash(attrs))
+    header = header + syntaxHighlighter['$docinfo']('head', node, Opal.hash(attrs))
   }
-  return ''
+  if (syntaxHighlighter !== Opal.nil && syntaxHighlighter['$docinfo?']('footer')) {
+    // reordering all link elements from footer to header, as pagedjs would otherwise render them too late in the process
+    let footer = syntaxHighlighter['$docinfo']('footer', node, Opal.hash(attrs))
+    // <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/highlight.js/9.15.6/styles/github.min.css"/>
+    const matches = footer.match(/(<link rel="stylesheet"[^>]*\/>)/g)
```

Comment:
We should extract this regular expression as a constant.
You can keep the example, so we know what the regular expression should match.

Reply:
fixed in next push.